### ISSUE TITLE
Benchmarks: Make source code for the benchmark dashboard public

### ISSUE THREAD

**Issue by divega** [dotnet/efcore#8285]
We internally use a Web based interface @rowanmiller wrote to look at our benchmark results. It should the delta between EF6 and EF Core and charts for the different scenarios. If there is enough interest (there was a bit in https://github.com/aspnet/EntityFramework/issues/8170#issue-221802029) we can consider getting this code in this repo (or in some other repo) so anyone and run it and improve it.

**Comment by ajcvickers** [comment]
Everyone: please thumbs-up if you want us to do this.

**Comment by divega** [comment]
Clearing up milestone. I think this issue is obsolete.

**Comment by davidroth** [comment]
@divega is there a public site for accessing the benchmark results? (Performance timeline)

**Comment by divega** [comment]
@davidroth when we switched to Benchmark.NET we changed the scheme we use to store the results and our custom dashboard became obsolete. That is why this issue became obsolete. 

We have recently started to using Power BI to visualize certain data useful to us, e.g. to identify any regressions in 2.1. But, fair enough, we haven’t published the results to the web and we should probably have that as a goal. 

But I believe before we do it, we should also come up with standard set of views that show relevant data that customers want to see/compare.

Perhaps you already have some ideas of what you want to see. In that case feel free to create a new issue specifically for that.
